src/instruments/view/instrumentlistmodel.cpp

```diff
@@ -700,3 +646,8 @@ void InstrumentListModel::makeCustomizedScoreOrder(const ScoreOrderInfo& order)
     m_selectedScoreOrderIndex = indexOfScoreOrderId(order.id);
     m_scoreOrders.insert(m_selectedScoreOrderIndex, customizedOrder);
 }
+
+bool InstrumentListModel::isInsrumentIndexValid(int index) const
```

Comment:
Typo, missing `t` in `isInsrumentIndexValid`?

Reply:
fixed